src/router/Routes.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+import Home from "../Containers/HomePage";
+import Fav from "../Containers/FavPage";
+import Search from "../Containers/SearchPage";
+
+const Routes = () => {
```

Comment:
Try to change into dynamic routing mindset, in this case every route is at the same index but defining your routes on a separate file before anything renders is reinforcing static practices.

As your application grow larger it might not make a lot of sense to isolate routes this way.

Reply:
:D I will work on that. Thanks for your feedback